sdk/search/tests.yml

```diff
@@ -0,0 +1,15 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+    parameters:
+      ServiceDirectory: search
+      DisableAzureResourceCreation: true
+      Timeout: 120
+      maxParallel: 3
```

Comment:
Nit: This should be `MaxParallel` ... Is there a reason to restrict parallelism to a max of 3?

Reply:
Good catch, this is being limited since we only have a limited pool of Search resources for all of our tests (other languages included) and wanted to restrict how many concurrent Java runs are happening at once.

Reply:
Search has quota on services can be created per subscription id + region at a time. Since all languages are using on the same quota, the test might failed by exhausts the quota. We limit the parallel run here for temporarily mitigating the issue.